.github/workflows/test.yaml

```diff
@@ -0,0 +1,50 @@
+name: Run test pull request
+
+on:
+    push:
+        branches:
+            - bas
+
+jobs:
+    test:
+        name: Course compose CI/CD testing
+        runs-on: ubuntu-latest
+        env:
+            DATABASE_URL: ${{ secrets.DATABASE_URL }}
```

Comment:
database url is considered to be one of the most sensitive data since it gives direct access to our databases.

we can use Github's Secret feature to prevent it from being seen by the public.

ref:
https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions

Reply:
I already rebased and removed all the commits that the database url was leaking. No worries, it's gone :)